wpeverest/user-registration
# UR-1359 Enhancement Text Area Enhancement

### All Submissions:






### Changes proposed in this Pull Request:



Previously, there is not present of total words/characters count and inputted words/characters count below textarea field in the registration form. This PR enhances above feature.
Closes # .

### How to test the changes in this Pull Request:

1. Go to form builder. Choose textarea from the form builder.  Select field options.
2. Go to advance setting. Toggle on the limit length. Set limit count and limit mode. Update the form.
3. Preview the form.
4. Now below the textarea word/character count appears.

### Types of changes:




### Other information:




### Changelog entry

> Enhancement Text Area Enhancement
